src/components/views/auth/AuthBody.tsx

```diff
@@ -17,6 +17,16 @@ limitations under the License.
 import React from 'react';
 
 export default class AuthBody extends React.PureComponent {
+    componentDidMount() {
+        const unloadCallback = (event) => {
+            event.preventDefault();
+            event.returnValue = "";
+            return "";
```

Comment:
Is it necessary to set these two values? [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Window/beforeunload_event) claims that calling `preventDefault` is enough.

Reply:
> Is it necessary to set these two values? [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Window/beforeunload_event) claims that calling `preventDefault` is enough.


I just read MDN and stack overflow why that is necessary, it's for legacy reasons, without setting the string and returning it, `beforeunload` will not work correctly in some browsers, and yes i tried removing both but it didn't bring the dialog on my chrome browser. so i suppose it's necessary for browser's backward compatibility.